include/ebpf_api.h

```diff
@@ -578,6 +605,48 @@ extern "C"
     ebpf_ring_buffer_map_write(
         fd_t ring_buffer_map_fd, _In_reads_bytes_(data_length) const void* data, size_t data_length) EBPF_NO_EXCEPT;
 
+    /**
+     * @brief Get eBPF program type for the specified BPF program type.
+     *
+     * @param[in] program_type BPF program type.
+     *
+     * @returns Pointer to eBPF program type, or NULL if not found.
+     */
+    _Ret_maybenull_ const ebpf_program_type_t*
+    ebpf_get_ebpf_program_type(bpf_prog_type_t bpf_program_type) EBPF_NO_EXCEPT;
+
+    /**
+     * @brief Get eBPF attach type for the specified BPF attach type.
+     *
+     * @param[in] bpf_attach_type BPF attach type.
+     * @param[out] ebpf_attach_type eBPF attach type or GUID_NULL.
+     *
+     * @returns EBPF_INVALID_ARGUMENT if attach type is unknown, EBPF_SUCCESS otherwise.
```

Comment:
```suggestion
     * @retval EBPF_SUCCESS The operation was successful.
     * @retval EBPF_INVALID_ARGUMENT The attach type is unknown.
```

Reply:
Can I address this in a follow up? Collecting reviews is a bit painful :sweat_smile:

Reply:
Yes, I already approved this PR :)